File: servicces/src/doan.ProjectManagement.Application.Contracts/ProjectTasks/Dto/ProjectTaskDto.cs
using doan.ProjectManagement.Enum;
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.ProjectTasks.Dto
{
    public class ProjectTaskDto : EntityDto<Guid>
    {
        public Guid? ProjectInformationId { get; set; }
        public string Name { get; set; }
        public string Descriptions { get; set; }
        public TaskType Type { get; set; }
        public string TeacherName { get; set; }
        public string Bonus { get; set; }
    }
}

File: applications/admin-web/src/app/shared/services/project/project-task/models/project-task-dto.ts
import { EntityDto } from '@abp/ng.core';
import { TaskType } from './task-type';

export class ProjectTaskDto extends EntityDto<string>{
  projectInformationId: string;
  name: string;
  descriptions: string;
  type: TaskType;
  teacherName: string;
  bonus: string;

  constructor(initial: Partial<ProjectTaskDto> = {}) {
    super(initial);
  }
}
